<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DerivationDialog</class>
 <widget class="QDialog" name="DerivationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Derive a file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_derivationChk">
     <property name="title">
      <string>Derive this file</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>This dialog is open when a file is standard or when a new file is saved. You can define the directory and the prefix to use for the new file.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Derivation Path :</string>
        </property>
        <property name="buddy">
         <cstring>m_derivationPathList</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="m_derivationPathList"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Prefixes :</string>
        </property>
        <property name="buddy">
         <cstring>m_prefixList</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="m_prefixList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;New file name :</string>
     </property>
     <property name="buddy">
      <cstring>m_directoryEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DirectoryEditWidget" name="m_directoryEdit" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEditWidget</class>
   <extends>QWidget</extends>
   <header location="global">directoryedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DerivationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DerivationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
